modules/pensions/lib/pensions/pdf_fill/va21p527ez.rb

```diff
@@ -1476,10 +1476,10 @@ def expand_pension_information
         )
 
         # If "YES," skip question 4B
-        @form_data['medicalCondition'] = 'Off' if @form_data['socialSecurityDisability'] == 1
+        @form_data['medicalCondition'] = nil if @form_data['socialSecurityDisability'].zero?
 
         # If "NO," skip question 4D
-        @form_data['medicaidStatus'] = 'Off' if @form_data['nursingHome'] == 2
+        @form_data['medicaidStatus'] = nil if @form_data['nursingHome'] == 1
```

Comment:
HexaPDF doesn't like us using `'Off'` here and prefers `nil` for a unchecked radiobutton